### PR TITLE
TST: parametrize slow test

### PR DESCRIPTION
**Pull request by jbrockmendel**

### CODE REVIEW

File: pandas/tests/indexing/multiindex/test_indexing_slow.py [hunk 1]
```diff
@@ -7,17 +7,46 @@
 from pandas import DataFrame, Series
 import pandas._testing as tm
 
+m = 50
```
Comment (jreback) [inline pandas/tests/indexing/multiindex/test_indexing_slow.py:10]: shouldnt u do this in a function where u set the random seed? not sure it's a big deal

**Review by jreback — CHANGES_REQUESTED** [review]

File: pandas/tests/indexing/multiindex/test_indexing_slow.py (hunk 1 above)
Comment (jbrockmendel) [reply pandas/tests/indexing/multiindex/test_indexing_slow.py:10]: we dont set randomseed in this module.  i dont think it makes  real difference

Comment (jreback) [reply pandas/tests/indexing/multiindex/test_indexing_slow.py:10]: right but this should still be in a function and just create a fixture out of it

Comment (jreback) [reply pandas/tests/indexing/multiindex/test_indexing_slow.py:10]: actually this is fine, i realize its the only thing in this file.

**Review by jreback — APPROVED** [review]

**Pull request merged**